src/components/CDPList.js

```diff
@@ -93,9 +95,10 @@ const CDPList = memo(function({ currentPath, viewedAddress, currentQuery }) {
           />
         );
       })}
-      {!account ? null : (
+      {!showPlusButton ? null : (
         <DashedFakeButton
           onClick={() =>
+            account &&
             show({ modalType: 'cdpcreate', modalTemplate: 'fullscreen' })
```

Comment:
The design shows a + button to create a cdp, even when on `/borrow`. Not that important but we may want to retract this

Reply:
Good point--better to hide the button entirely